src/NzbDrone.Core/Notifications/Trakt/Resource/TraktListResource.cs

```diff
@@ -0,0 +1,24 @@
+using Newtonsoft.Json;
+
+namespace NzbDrone.Core.Notifications.Trakt.Resource
+{
+    public class TraktListResource
+    {
+        public int? Rank { get; set; }
+        [JsonProperty(PropertyName = "listed_at")]
+        public string ListedAt { get; set; }
+        public string Type { get; set; }
```

Comment:
Nit: I'd group up all the props that don't have decorators (`Type`, `Watchers` and `Revenue`)